examples/tree_2d_dgsem/elixir_navierstokes_shearlayer_amr.jl

```diff
@@ -7,12 +7,20 @@ using Trixi
 
 # TODO: parabolic; unify names of these accessor functions
 prandtl_number() = 0.72
-mu() = 1.0/3.0 * 10^(-3) # equivalent to Re = 3000
+mu() = 1.0/3.0 * 10^(-5)
```

Comment:
I added this file in https://github.com/trixi-framework/Trixi.jl/pull/1591/files

but never really used it - now it serves as a neat motivation for velocity-based AMR